targets/qtpy.json

```diff
@@ -0,0 +1,8 @@
+{
+    "inherits": ["atsamd21e18a"],
+    "build-tags": ["sam", "atsamd21e18a", "qtpy"],
```

Comment:
The build tags could be simpler, see #1490 (fixing this for all other SAM boards too).